shared/services/config/smartnode-config.go

```diff
@@ -622,16 +336,28 @@ func (cfg *SmartnodeConfig) GetParameters() []*config.Parameter {
 
 // Getters for the non-editable parameters
 
+// Gets the currently chosen network's configuration
+func (cfg *SmartnodeConfig) GetNetworkInfo() *config.NetworkInfo {
+	for _, network := range cfg.parent.networks.GetNetworks() {
+		networkName := fmt.Sprintf("%s", cfg.Network.Value)
```

Comment:
hmm, when we parse the .yml files maybe we should save them in a `map[config.Network]NetworkInfo` that we can access here without the loop.